app/controllers/api/storage.php

```diff
@@ -1067,7 +878,7 @@
         $fileLogos = Config::getParam('storage-logos');
 
         $date = \date('D, d M Y H:i:s', \time() + (60 * 60 * 24 * 45)).' GMT';  // 45 days cache
-        $key = \md5($fileId.$width.$height.$gravity.$quality.$borderWidth.$borderColor.$borderRadius.$opacity.$rotation.$background.$storage.$output);
```

Comment:
A piece of functionality was removed here, just double-checking if that is okay.